src/components/ChatEntry.js

```diff
@@ -1,22 +1,34 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp'
+
+const ChatEntry = (singleEntry) => {
```

Comment:
There's nothing wrong with naming this incoming prop `singleEntry` but just remember that conventionally you will find it named `props`.